quicksort/quick_sort.py

```diff
@@ -0,0 +1,44 @@
+def partition(array, p, r):
+    """
+    Perform Partition Operation on array.
+    Time Complexity: Theta(nLogn)
+    Auxiliary Space: O(n)
+    :param array: Iterable of elements
+    :param p: pivot value for array
+    :param r: right limit of array
+    :return: return q value for function, used in partitioning of array.
+    """
+    i = p - 1
+    pivot = array[r]
+    for j in range(p, r):
+        if(array[j] <= pivot):
+            i = i + 1
+            array[i], array[j] = array[j], array[i]
+    i = i + 1
+    array[r], array[i] = array[i], array[r]
+    return i
+
+
+def quick_sort(array, pivot, right):
+    """
+    Perform sort using partition function.
+    Time Complexity : O(nlog(n)).
+    Space Complexity : O(n).
+    :param array: Iterable of elements
+    :param pivot: used as left limit of quick sort
+    :param right: right limit for quick sort
+    :return: no returns, sorts array
+    """
+    if(pivot < right):
```

Comment:
`if pivot < right:` would look more pythonic as in Python we can afford to drop brackets in conditions. Same for the other `if` statement also.